runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/components.json

```diff
@@ -89,6 +89,9 @@
     "Component.HistoryTable.Rows.Initiator.Api": "API({{ login }})",
     "Component.HistoryTable.Rows.Initiator.Email": "Email({{ email }})",
 
+    "Component.HistoryTable.Special.LoadMore": "Załaduj kolejne {{ pageSize }} (pozostało {{ count }})",
```

Comment:
Not sure why we provide here 2 numbers, do we need pageSize here?. Maybe just "Załaduj więcej (pozostało {{ count }})"? it will be also grammatically correct for 1.

Reply:
in case there are over 100 processes there's pagination for loading them, the load all could be overkill. for under 100 is just "load more". How does that sounds to you?